### ISSUE TITLE
head(){return { script:[...]}} doesnt inject <script> in the first load.

### ISSUE THREAD

**Issue by woodgates** [nuxt/nuxt.js#3872]
### Version

[v1.4.2](https://github.com/nuxt.js/releases/tag/v1.4.2)

### Reproduction link

[https://github.com/woodgates/nuxtadochat](https://github.com/woodgates/nuxtadochat)

### Steps to reproduce

Hi.
I'm trying to make a simple chat app with Nuxt, adoniis and Vuetify. The issue I found is that in my chat.vue page I got this scripts:
```
  head () {
    return {
      script:[
        {src: "https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/6.26.0/polyfill.min.js"},
        {hid:'adonis', src: "https://unpkg.com/@adonisjs/websocket-client@1.0.9/dist/Ws.browser.js"}
      ]
    } 
```

One the first page load I got this error: "adonis is not defined".

When I check the elements of the page I found that both scripts were not in the head of the page. When I reload the page the scripts appear on the head:
```
<script data-n-head="true" src="https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/6.26.0/polyfill.min.js"></script>
<script data-n-head="true" data-hid="adonis" src="https://unpkg.com/@adonisjs/websocket-client@1.0.9/dist/Ws.browser.js"></script>
```
Every time I open chromium for the first time I got this error.

Another issue related to this is that when I change the page and the I return to the chat page I got this error:
```
polyfill.min.js:1 Uncaught Error: only one instance of babel-polyfill is allowed
```

 

### What is expected ?

The scripts should appear on the <head> since the first load

### What is actually happening?

scripts are injected only after reloading the page.

### Additional comments?

You can reproduce the problem cloning the repo.

<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c7718">#c7718</a>)</em></sub></div>

**Comment by manniL** [comment]
Duplicate of #3396 
See my last comment there
